docs/trino-connector/catalog-hive.md

```diff
@@ -302,3 +302,47 @@ replacing hdfs_user with the appropriate username:
 ```text
 -DHADOOP_USER_NAME=hdfs_user
 ```
+
+## S3
+
+When using AWS S3 within the Hive catalog, users need to configure the Trino Hive connector's
+AWS S3-related properties in the catalog's properteis. Please refer to the documentation
+of [Hive connector with Amazon S3](https://trino.io/docs/435/connector/hive-s3.html).
+
+To create a Hive catalog with AWS S3 configuration in the Trino CLI, use the following command:
+
+```sql
+call gravitino.system.create_catalog(
+  'gt_hive',
+  'hive',
+  map(
+    array['metastore.uris',
+        'trino.bypass.hive.s3.aws-access-key', 'trino.bypass.hive.s3.aws-secret-key', 'trino.bypass.hive.s3.region'
+    ],
+    array['thrift://hive:9083', '<aws-access-key>', '<aws-secret-key>', '<region>']
+  )
+);
+```
+
+- The settings for `trino.bypass.hive.s3.aws-access-key`, `trino.bypass.hive.s3.aws-secret-key` and `trino.bypass.hive.s3.region`
```

Comment:
could trino support different catalogs with different secret keys?

Reply:
The Trino supported.